networker/networker.go

```diff
@@ -1138,13 +1167,25 @@ func (netw *Combined) setMesh(
 		if err = netw.policyRouter.SetupRoutingRules(
 			netw.mesh.Tun().Interface(),
 			false,
+			netw.routeToRemoteLan,
```

Comment:
This should always be false and the else is not necessary, because we are still connected to VPN and the suppress prefix rule will never change.

Reply:
right, this `else` part is not logical, but current logic would create duplicate fwMark rules during network change event